cmd/query/app/apiv3/gateway_test.go

```diff
@@ -143,9 +144,8 @@ func (gw *testGateway) runGatewayGetOperations(t *testing.T) {
 }
 
 func (gw *testGateway) runGatewayGetTrace(t *testing.T) {
-	query := tracestore.GetTraceParams{TraceID: traceID}
 	gw.reader.
-		On("GetTraces", matchContext, query).
+		On("GetTraces", matchContext, mock.AnythingOfType("[]tracestore.GetTraceParams")).
```

Comment:
why match in string name? The error was that the function was expecting an array, so this should've worked:
```
	query := []tracestore.GetTraceParams{{TraceID: traceID}}
	gw.reader.
		On("GetTraces", matchContext, query).
```